src/scenes/Partners/Detail/PartnerDetail.tsx

```diff
@@ -60,21 +113,98 @@ export const PartnerDetail = () => {
                 <Skeleton width="75%" />
               )}
             </Typography>
-            {/* {partner ? (
+            <Tooltip title="Update Global Innovations Client">
               <IconButton
                 color="primary"
                 aria-label="edit partner"
-                onClick={editPartner}
+                onClick={() => editPartner('globalInnovationsClient')}
               >
                 <PencilCircledIcon />
               </IconButton>
-            ) : null} */}
+            </Tooltip>
+          </header>
+          <div className={classes.subheader}>
+            <Typography variant="h4">
+              {partner ? 'Partner Information' : <Skeleton width={200} />}
+            </Typography>
+            {partner && (
+              <Typography variant="body2" color="textSecondary">
+                Created {formatDate(partner.createdAt)}
+              </Typography>
+            )}
+          </div>
+          <Grid container spacing={1} alignItems="center">
+            <Grid item>
+              <DataButton
+                onClick={() => editPartner('pmcEntityCode')}
+                secured={partner?.pmcEntityCode}
+                redacted="You do not have permission to view PMC Entity Code"
+                children={`PMC Entity Code${
+                  partner?.pmcEntityCode.value
+                    ? `: ${partner.pmcEntityCode.value}`
+                    : ''
+                }`}
+              />
+            </Grid>
+            <Grid item>
+              <DataButton
+                onClick={() => editPartner('active')}
+                secured={partner?.active}
+                redacted="You do not have permission to view Status"
+                children={partner?.active.value ? 'Active' : 'Inactive'}
+              />
+            </Grid>
+            <Grid item>
+              <BooleanProperty
+                label="Global Innovations Client"
+                redacted="You do not have permission to view whether this is a Global Innovations Client"
+                data={partner?.globalInnovationsClient}
+                wrap={(node) => <Grid item>{node}</Grid>}
+              />
+            </Grid>
+          </Grid>
+          <Grid container spacing={3}>
+            <Grid item xs={6} className={classes.cardSection}>
+              <Typography variant="h3">Partner Types</Typography>
```

Comment:
It's weird that these don't have loading states. The scene type even does ("Partner")

Reply:
Added loading states, this is what the loading screen looks like now:
<img width="1314" alt="Screen Shot 2020-10-20 at 1 32 37 PM" src="https://user-images.githubusercontent.com/43487134/96641997-59946500-12da-11eb-94e9-73c94d1106df.png">